app/lib/features/news/widgets/news_full_view.dart

```diff
@@ -25,9 +25,18 @@ class _NewsVerticalViewState extends ConsumerState<NewsFullView> {
   @override
   void initState() {
     super.initState();
-    _pageController = PageController(
-      initialPage: widget.initialPageIndex,
-    );
+    _pageController = PageController(initialPage: widget.initialPageIndex);
+    //LISTEN for boost index
+    ref.listenManual(boostGoToPageProvider, fireImmediately: true,
```

Comment:
`ref.read(boostGoToPageProvider.notifier).state = 0;` supposed to trigger this listenManual Function but it is not.
@gnunicorn